src/main/java/com/devcycle/sdk/server/cloud/api/DevCycleCloudClient.java

```diff
@@ -69,9 +76,9 @@ public Map<String, Feature> allFeatures(DevCycleUser user) throws DevCycleExcept
    * @param defaultValue Default value to use if the variable could not be fetched
    *                     (required)
    * @return Variable value
-   * @throws DevCycleException If there are any uses with the data provided
+   * @throws IllegalArgumentException If there are any issues with the key or default value provided
    */
-  public <T> T variableValue(DevCycleUser user, String key, T defaultValue)  throws DevCycleException {
+  public <T> T variableValue(DevCycleUser user, String key, T defaultValue)  {
```

Comment:
variableValue() and variable() should not be throwing an exception that forces the user to catch it, especially when the exception is only used during parameter validation.  I updated it to use IllegalArgumentException to match what DevCycleLocalClient does